CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,24 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    public var titleFont: UIFont = UIFont.systemFontOfSize(16) {
+        didSet {
+            updateTabs()
+        }
+    }
+    
     @IBInspectable public var lightBackground: Bool = true {
         didSet {
             updateBackground()
         }
     }
     
+    @IBInspectable public var darkBackground: Bool = false {
```

Comment:
Maybe instead do a `translucentBackground` property and use that. 

> On Apr 21, 2016, at 2:00 PM, Parker Rushton <notifications@github.com> wrote:
> 
> In CustomTabBar/CustomTabBar.swift:
> 
> >      @IBInspectable public var lightBackground: Bool = true {
> >          didSet {
> >              updateBackground()
> >          }
> >      }
> >      
> > +    @IBInspectable public var darkBackground: Bool = false {
> Correct
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>